stdlib/Markdown/src/render/latex.jl

```diff
@@ -167,6 +167,16 @@ function latexesc(io, s::AbstractString)
     end
 end
 
+"""
+    latex([io::IO], md)
+Convert a Markdown object to a LaTeX expression. If no `io` is provided, the result will be returned as a `String`.
```

Comment:
```suggestion
```suggestion
show(io::IO, ::MIME"text/latex", md::MD) = latex(io, md)

"""
    latex([io::IO], md)

Output the contents of the Markdown object `md` in LaTeX format, either
writing to an (optional) `io` stream or returning a string.

One can alternatively use `show(io, "text/latex", md)` or `repr("text/latex", md)`.
``````